hera_cal/smooth_cal.py

```diff
@@ -667,8 +808,9 @@ def freq_filter(self, filter_scale=10.0, tol=1e-09, window='tukey', skip_wgt=0.1
                     self.flag_grids[ant][i, :] = np.ones_like(self.flag_grids[ant][i, :])
         self.rephase_to_refant(warn=False)
 
-    def time_freq_2D_filter(self, freq_scale=10.0, time_scale=1800.0, tol=1e-09,
-                            filter_mode='rect', window='tukey', maxiter=100, **win_kwargs):
+    def time_freq_2D_filter(self, freq_scale=10.0, time_scale=1800.0, tol=1e-09, filter_mode='rect',
```

Comment:
In general, this function doesn't have enough inline comments to explain what's going on.

Reply:
I added more comments and simplified the function. Let me know if you'd like for me to add more!